### PR TITLE
Packaging: adding make file for gstatus package build

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
tendrl-bug-id: Tendrl/gluster-integration#151
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat , @shtripat , please review

File: dependencies/gstatus/Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+VERSION=0.64
+
+clean:
+	rm -fr gstatus*
+
+dist:
+	git clone https://github.com/gluster/gstatus > /dev/null || : \
+	&& cd gstatus \
+	&& python setup.py sdist \
+	  && mv dist/$(NAME)-$(VERSION).tar.gz ../. \
```
Comment (r0h4n) [inline dependencies/gstatus/Makefile]: Use https://github.com/gluster/gstatus/archive/v$VERSION.tar.gz

Comment (TimothyAsir) [inline dependencies/gstatus/Makefile]: Done

**Comment by TimothyAsir** [comment]
@r0h4n please review

**Pull request merged**